libs/datamodel/core/tests/attributes/id_positive.rs

```diff
@@ -304,18 +304,101 @@ fn id_accepts_length_arg_on_mysql() {
     );
 
     let schema = parse(&dml);
-    schema.assert_has_model("User");
-
-    // user_model.assert_has_pk(PrimaryKeyDefinition {
-    //     name: None,
-    //     db_name: Some("User_pkey".to_string()),
-    //     fields: vec![
-    //         ("firstName".to_string(), None),
-    //         ("middleName".to_string(), Some(1)),
-    //         ("lastName".to_string(), None),
-    //     ],
-    //     defined_on_field: false,
-    // });
+    let user_model = schema.assert_has_model("User");
+    let blog_model = schema.assert_has_model("Blog");
+
+    user_model.assert_has_pk(PrimaryKeyDefinition {
```

Comment:
I'd like these to be more granular... e.g.

```rust
user_model.assert_has_pk().assert_contains_field(0).assert_references("firstName").assert_sort_order(Asc)
```

We don't need to do that now though.